Unknown tagged block b'vowv'

I've been getting this warning and saw back in issue [#63 ](https://github.com/psd-tools/psd-tools/issues/63) that a similar warning was fixed by adding the key and parsing function to constants.py and layer_effects.py. What I'm curious about is how to find the formatting of the undocumented tagged block and how to compare it to the others so that I can determine if that is going to be a viable approach to solve this issue.

@kyamagu Thanks, for the quick response. I tried out both DescriptorBlock and DescriptorBlock2 but both resulted in assertion errors. DescriptorBlock resulted in `AssertionError: read=0, expected=4` and DescriptorBlock2 resulted in `AssertionError: read=4, expected=8` I then tried EmptyElement and was able to parse the .psd file with no warnings or errors, and the output was as expected. Though I have a feeling that EmptyElement isn't the right format to parse the binary and this is really just a bandaid fix to get the warnings to go away... That's just a hunch though as I am still very new to python.

--- a/src/psd_tools/constants.py
+++ b/src/psd_tools/constants.py
@@ -373,6 +373,7 @@ class Tag(bytes, Enum):
     VECTOR_MASK_SETTING1 = b'vmsk'
     VECTOR_MASK_SETTING2 = b'vsms'
     VECTOR_ORIGINATION_DATA = b'vogk'
+    VECTOR_ORIGINATION_UNKNOWN = b'vowv'
     VECTOR_STROKE_DATA = b'vstk'
     VECTOR_STROKE_CONTENT_DATA = b'vscg'
     VIBRANCE = b'vibA'

--- a/src/psd_tools/psd/tagged_blocks.py
+++ b/src/psd_tools/psd/tagged_blocks.py
@@ -93,6 +93,7 @@ TYPES.update({
     Tag.VECTOR_MASK_SETTING1: VectorMaskSetting,
     Tag.VECTOR_MASK_SETTING2: VectorMaskSetting,
     Tag.VECTOR_ORIGINATION_DATA: DescriptorBlock2,
+    Tag.VECTOR_ORIGINATION_UNKNOWN: IntegerElement,
     Tag.VECTOR_STROKE_DATA: DescriptorBlock,
     Tag.VECTOR_STROKE_CONTENT_DATA: VectorStrokeContentSetting,
 })

--- a/tests/psd_tools/psd/test_tagged_blocks.py
+++ b/tests/psd_tools/psd/test_tagged_blocks.py
@@ -50,6 +50,7 @@ def test_tagged_blocks_v2():
         (Tag.LAYER_VERSION, IntegerElement(1), 2, 1),
         (Tag.LAYER_VERSION, IntegerElement(1), 1, 4),
         (Tag.LAYER_VERSION, IntegerElement(1), 2, 4),
+        (Tag.VECTOR_ORIGINATION_UNKNOWN, IntegerElement(2), 2, 1),
     ]
 )
 def test_tagged_block(key, data, version, padding):

